keyboards/cipulot/kawayo/rules.mk

```diff
@@ -0,0 +1,20 @@
+# MCU name
+MCU = STM32F411
+
```

Comment:
```suggestion

BOARD = GENERIC_STM32_F411XE

```

Reply:
Added suggested line to the file. Tested working with the board proto.